doc/source/user_guide/io.rst

```diff
@@ -5477,6 +5478,42 @@ web site.
 
 No official documentation is available for the SAS7BDAT format.
 
+.. _io.spss:
+
+.. _io.spss_reader:
+
+SPSS formats
+------------
+
+The top-level function :func:`read_spss` can read (but not write) SPSS
+`sav` (.sav) and  `zsav` (.zsav) format files(since *v0.25.0*).
+
+SPSS files contain column names. By default the
+whole file is read, categorical columns are converted into ``pd.Categorical``
+and a ``DataFrame`` with all columns is returned.
+
+Specify a ``usecols`` to obtain a subset of columns. Specify ``convert_categoricals=False``
+to avoid converting categorical columns into ``pd.Categorical``.
+
+Read a spss file:
+
+.. code-block:: python
+
+    df = pd.read_spss('spss_data.zsav')
+
+Extract a subset of columns ``usecols`` from SPSS file and
+avoid converting categorical columns into ``pd.Categorical``:
+
+.. code-block:: python
+
+    df = pd.read_spss('spss_data.zsav', usecols=['foo', 'bar'],
+                      convert_categoricals=False)
+
+More info_ about the sav and zsav file format is available from the IBM
```

Comment:
Any chance you can share a screenshot of what this looks like when this is built? I'm not a sphinx expert just want to make sure this renders appropriately

Reply:
It's just a hyperlink. If it's a single word you don't need the backpacks.

Reply:
Thanks for the tips! Learned Sphinx yesterday and it is more fun than Markdown file.

Reply:
@WillAyd I put the final result in the top. Have a look~